packages/shell-api/src/field-level-encryption.ts

```diff
@@ -113,6 +120,7 @@ export class KeyVault extends ShellApiClass {
 
   @returnsPromise
   createKey(
+    // TODO kms cannot be a document
```

Comment:
See this conversation for the context: https://docs.google.com/document/d/1v0UUDvK4Ch8WM03XAFw1Ci-hem84vrgIw07RWCwmyJ4/edit?disco=AAAALAO7L-U